lib/board.rb

```diff
@@ -13,6 +13,9 @@ def width
     @grid.first.length
   end
 
+  def paint!(row, col)
+    @grid[row][col].paint!
```

Comment:
I had to learn that `[row][col]` is the crossroads for the grid location (aka, the cell we're going to paint). I'm so used to thinking of grids differently (more like `[[]]` than `[][]`) that I didn't see what was happening initially. Just commenting on this for my own retention.

Reply:
It's a good thing to note! This is a common method for indexing into 2-dimensional arrays. I assume it's the most common.